services/ui-src/src/routes/Dashboard/Dashboard.tsx

```diff
@@ -159,6 +160,11 @@ export const Dashboard = () => {
             />
           )
         ) : (
+          <Flex alignItems="center" w="full" justifyContent="center" p="10">
```

Comment:
Nit: Any more than 1 or 2 inline styles and it's usually a good sign you want to make a class or `sx` entry for readability.